src/__tests__/mass.test.ts

```diff
@@ -1,71 +1,71 @@
-import { readFileSync } from 'fs';
+import { readFileSync } from "fs";
 
-import { parseXY } from '..';
+import { DataXY, parseXY } from "..";
 
 const path = `${__dirname}/../../testFiles/`;
 
-describe('Test mass spectra', () => {
-  it('mass1.txt', () => {
-    let filename = 'mass1.txt';
+describe("Test mass spectra", () => {
```

Comment:
Should normally auto-format to single quotes. Please add .prettierrc (as you did in global-spectra-deconvolution)

Reply:
added